tests/test_translator.py

```
@@ -57,8 +58,8 @@ def test_init(openai_client):
     assert "all_critiques" in translator.translation_log
 
 
-def test_translate_text(translator_instance):
-    """Test the translate_text method."""
+def test_translate_text_without_context(translator_instance):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_translate_text_without_context`

Add return type annotation: `None`

(ANN201)